pages/web/domains/dns_zone_create/guide.en-gb.md

```diff
@@ -1,70 +1,122 @@
+Once the verification is complete, choose whether to enable the minimum records for the DNS zone you are going to create. This choice is not definitive, as you can always [edit DNS zone records](/pages/web/domains/dns_zone_edit) later.
 
 ![dnszonecreate](images/dns-zone-create-step2.png){.thumbnail}
 
-Once you have selected an option, continue following the next steps until you have created the DNS zone.
+|Enable minimal entries?|Details|
+|---|---|
+|Yes|Select this option if you would like to customise the DNS zone yourself at a later stage.</br>![minimal-dns-entries](images/minimal.png){.thumbnail}|
+|No|Select this option if you plan to use OVHcloud services as a [web hosting](https://www.ovhcloud.com/en-gb/web-hosting/){.external}, with the zone preconfigured for this purpose.</br>![no-minimum-dns-entries](images/no_minimal.png){.thumbnail}|
+
+Once you have chosen your DNS zone, follow the steps until you have created it.
 
 ### Step 2: Edit the DNS zone (optional)
 
-Now that your domain name’s DNS zone has been created, you can edit it. This step is optional, but it may be essential if you want to ensure that any services linked to your domain name do not experience any downtime (e.g. your website and email services).
+Now that your domain name’s DNS zone has been created, you can edit it. This step is optional, but it may be necessary if you want to ensure that the services associated with your domain name remain available (e.g. your website and/or emails).
 
-If you would like to edit this DNS zone, click `Web Cloud`{.action} in the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB), then click `Domain names`{.action} and choose the domain name concerned. Go to the `DNS zone`{.action} tab.
+To edit this DNS zone, please read our guide on [Editing a DNS zone at OVHcloud](/pages/web/domains/dns_zone_edit).
 
 > [!primary]
 >
-> If you have just created the DNS zone but the domain name does not appear under the list of services in the `Domain names`{.action} section, please wait a few moments, then reload the page.
+> If you have just created the DNS zone and the domain name does not yet appear in your list of services (under `Domain names`{.action} in the `Web Cloud`{.action} section of the OVHcloud Control Panel), wait 15 to 20 minutes, then reload the page.
 >
 
-Once it appears, make the required changes. To learn more about how to edit a DNS zone, please read our guide to [Editing an OVHcloud DNS zone](/pages/web/domains/dns_zone_edit){.external}. Once you have modified your domain name’s OVHcloud DNS zone, you will need to allow 4-24 hours for the changes to fully propagate and take effect.
+Once you have done this, make the required changes. Once you have modified your domain name’s OVHcloud DNS zone, you will need to allow between **4 and 24 hours** for the changes to propagate fully, and be effective.
```

Comment:
Why is this mentioned here? Propagation time does not matter at this point, since you only edited a new DNS zone. These records are not cached anywhere, propagation therefore irrelevant.

Reply:
It's right, I'll correct that from all subs. I don't know why I added that here.

Reply:
It's right, I'll correct that from all subs. I don't know why I added that here.

Reply:
resolved => commit => [cf42270]